Deploy/deployment.json

```diff
@@ -103,6 +116,18 @@
         "hostingEnvironment": "",
         "reserved": true
       }
+    },
+    {
+      "type": "Microsoft.CognitiveServices/accounts",
```

Comment:
You've added a conditional parameter `enablePersonalizer` but this resource is created no mater what. This resource should be conditional on the same boolean.

Reply:
this personalizer resource has a condition property that depends on enablePersonalizer, is there something further I'm missing?

Reply:
Nope you're not missing anything. Looks like, I missed the `condition` line.